infra/ballista/build/Dockerfile

```diff
@@ -0,0 +1,120 @@
+
+ARG BALLISTA_VERSION=0.8.0
+ARG RELEASE_FLAG=release
+ARG FUNCTION_DIR="/function"
+
+FROM debian:11.4 as ric-dependency
+
+# Install aws-lambda-cpp build dependencies
+RUN apt-get update && \
+    apt-get install -y \
+    g++ \
+    make \
+    cmake \
+    unzip \
+    python3 \
+    python3-pip \
+    libcurl4-openssl-dev
+
+# Include global arg in this stage of the build
+ARG FUNCTION_DIR
+# Create function directory
+RUN mkdir -p ${FUNCTION_DIR}
+
+# Copy function code
+COPY lambda-handler.py ${FUNCTION_DIR}
+
+# Install the runtime interface client
+RUN pip3 install \
+    --target ${FUNCTION_DIR} \
+    awslambdaric
+
+#ballista-builder.Dockerfile
+FROM rust:1.63.0-buster as builder
+
+# Include global arg in this stage of the build
+ARG RELEASE_FLAG
+ARG BALLISTA_VERSION
+
+ENV RUST_LOG=info
+ENV RUST_BACKTRACE=full
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get -y install libssl-dev openssl zlib1g zlib1g-dev libpq-dev cmake protobuf-compiler netcat curl unzip git
+
+# create build directories
+RUN mkdir -p /home/builder/workspace
+
+ENV HOME=/home/builder
+ENV PATH=$HOME/.cargo/bin:$PATH
+
+# prepare rust
+RUN rustup update && \
+    rustup component add rustfmt && \
+    cargo install cargo-chef --version 0.1.34
+
+#get ballista
+# if [${BALLISTA_VERSION}!= ""];
+# RUN curl -L "https://github.com/apache/arrow-ballista/archive/refs/tags/${BALLISTA_VERSION}.tar.gz"  | tar -xzf - -C /home/builder/workspace
+# if latest
```

Comment:
i think we can cleanup these commented snippets, they don't add much information

Reply:
ok. this comments were there to switch if we wanted to from latest code on master towards latest release.